client/boot/index.js

```diff
@@ -369,8 +369,8 @@ function reduxStoreReady( reduxStore ) {
 		require( 'lib/rubberband-scroll-disable' )( document.body );
 	}
 
-	if ( config.isEnabled( 'dev/test-helper' ) && document.querySelector( '.environment.is-tests ul.active-tests' ) ) {
-		require( 'lib/abtest/test-helper' )( document.querySelector( '.environment.is-tests ul.active-tests' ) );
+	if ( config.isEnabled( 'dev/test-helper' ) && document.querySelector( '.environment.is-tests' ) ) {
```

Comment:
I know it's better to be safe than sorry, but at the moment the `config.isEnabled( 'dev/test-helper' )` check is already required to have the DOM element present. Probably it's better to keep it as it is, but I thought it's worth mention it :)